app/controllers/AccountManagement.scala

```diff
@@ -314,5 +319,18 @@ object AccountManagement extends Controller with LazyLogging {
     }
   }
 
+  def renewThankYou() = accountManagementAction.async { implicit request =>
+    implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
+    implicit val tpBackend = resolution.backend
+    import model.SubscriptionOps._
+
+    val res = for {
+    subscription <- OptionT( SessionSubscription.subscriptionFromRequest)
+    billingSchedule <- OptionT(tpBackend.commonPaymentService.billingSchedule(subscription.id, numberOfBills = 13))
+    }yield {
+      Ok(thankyou_renew(subscription.latestPlan,billingSchedule, resolution))
```

Comment:
not really conventional to use _ in views, would normally be package.name.lowerCamelCase